src/Form/QuestionType/AbstractQuestionTypeSelectable.php

```diff
@@ -137,21 +175,23 @@ public function getValues(?Question $question): array
     }
 
     #[Override]
-    public function renderAdministrationTemplate(
-        ?Question $question = null,
-        ?string $input_prefix = null
-    ): string {
+    public function renderAdministrationTemplate(?Question $question = null): string
+    {
         $template = <<<TWIG
         {% set rand = random() %}
 
-        {% macro addOption(input_type, rand, checked, value, placeholder, extra_details = false, disabled = false) %}
-            {% set uuid = random() %}
+        {% macro addOption(input_type, checked, value, placeholder, uuid = null, extra_details = false, disabled = false) %}
+            {% if uuid is null %}
+                {% set uuid = random() %}
+            {% endif %}
 
             <div
                 class="d-flex gap-1 align-items-center mb-2"
                 {{ extra_details ? 'data-glpi-form-editor-question-extra-details' : '' }}
             >
                 <i
+                    role="button"
+                    aria-label="{{ __('Move option') }}"
```

Comment:
Translation in inline twig templates are not detected by our extraction tool.
We need to generate them in the PHP code and pass the value to the template.